src/tests/multihost/alltests/test_rfc2307.py

```diff
@@ -0,0 +1,107 @@
+""" Automation of rfc2307
+
+:requirement: rfc2307
+:casecomponent: sssd
+:subsystemteam: sst_idm_sssd
+:upstream: yes
+"""
+from __future__ import print_function
+import pytest
+from sssd.testlib.common.libkrb5 import krb5srv
+from sssd.testlib.common.utils import sssdTools, LdapOperations
+from constants import ds_instance_name, ds_suffix, ds_rootdn, ds_rootpw
+from sssd.testlib.common.exceptions import SSHLoginException, LdapException
+from sssd.testlib.common.expect import pexpect_ssh
+
+
+def usr_grp(multihost, obj_info, type):
+    """
+    Add an object, user or group, in the ldap-server
+        :param dict obj_info: an object(user/group) details
+        :param str type: Either 'user' or 'group'
+        :return: None
+        :exception: LdapException
+    """
+    ldap_uri = f'ldap://{multihost.master[0].sys_hostname}'
+    ldap_inst = LdapOperations(ldap_uri, ds_rootdn, ds_rootpw)
+    krb = krb5srv(multihost.master[0], 'EXAMPLE.TEST')
+    if type == 'user':
+        usr = obj_info.get('uid')
+        try:
+            if ldap_inst.posix_user("ou=People", ds_suffix, obj_info):
+                krb.add_principal(usr, 'user', 'Secret123')
+        except LdapException:
+            print(f"Unable to add ldap User {obj_info}")
+    if type == 'group':
+        try:
+            ldap_inst.posix_group("ou=Groups", ds_suffix, obj_info,
+                                  memberUid=obj_info.get('memberUid'))
+        except LdapException:
+            print(f"Unable to add ldap group {obj_info}")
+
+
+@pytest.mark.usefixtures('setup_sssd', 'create_posix_usersgroups')
+@pytest.mark.rfc2307
+class Testrfc2307(object):
+    """
+    This is test case class for ldap rfc2307
+
+        :setup:
+          1. Configure SSSD to authenticate against directory server
+          2. Enable debug_level to 9 in the 'nss', 'pam' and domain section
+    """
+    @pytest.mark.tier2
+    def test_0001_bz1362023(self, multihost, backupsssdconf):
+        """
+        :title: IDM-SSSD-TC: rfc2307: user with spaces at beginning
```

Comment:
This is porting, so are we using the same title and id from the polarion?
because we do have the same test case in polarion.

Reply:
I've used different ID and different title from original bash testcases.

Reply:
okay, so we are keeping bash test also?